lib/adminui.js

```diff
@@ -925,7 +925,9 @@ ADMINUI.prototype.createServer = function () {
             params: req.params
         });
 
-        res.send(error);
+        if (error.code !== 'ETIMEDOUT' || error.code !== 'ECONNRESET') {
```

Comment:
This error may fall during the request (if such a weak connection) and then called res.send twice which leads to crash application